sync-version/action.yaml

```diff
@@ -73,7 +71,7 @@ runs:
         yaml_path: ${{ steps.yaml-path.outputs.yaml_path }}
         prev_version: ${{ steps.yaml-path.outputs.prev_version }}
       env:
-        SNAP_NAME: ${{ inputs.snap-name || github.event.repository.name }}
+        SNAP_NAME: ${{ steps.yaml-path.outputs.snap-name || github.event.repository.name }}
```

Comment:
You can drop the OR here now and just rely on the output

Reply:
Yeah.. thought so, but just came up, like why not :-)